build-system/server/new-server/transforms/scripts/scripts-transform.ts

```diff
@@ -15,21 +15,28 @@
  */
 
 import {PostHTML} from 'posthtml';
-import {URL} from 'url';
-import {isValidScript} from '../utilities/script';
+import {isJsonScript, isValidScript, tryGetURL} from '../utilities/script';
 import {CDNURLToLocalDistURL} from '../utilities/cdn';
 import {OptionSet} from '../utilities/option-set';
+import {parse} from 'path';
 
 /**
  * For any script, with a valid path to AMP Project CDN, replace it with a local value.
  * @param script
  */
-function modifySrc(script: PostHTML.Node): PostHTML.Node {
-  if (!isValidScript(script)) {
+function modifySrc(script: PostHTML.Node, options: OptionSet): PostHTML.Node {
+  if (isJsonScript(script)) {
```

Comment:
This might benefit from some increased commenting, the order of the conditions if swapped would cause failures right?

Reply:
@kristoferbaxter thats true. i'll add

Reply:
added in both location this is used